ngraph/core/src/pass/constant_folding_scatter.cpp

```diff
@@ -108,6 +108,8 @@ static shared_ptr<op::Constant>
                      axis_type);
         break;
     }
+
+    return shared_ptr<op::Constant>();
```

Comment:
@t-jankowski Please try to replace each `NGRAPH_CHECK` to `throw ngraph_error` with the appropriate message instead of returning empty Constant at the end of method. I hope that compilator will accept it ;)

Reply:
Replacing `NGRAPH_CHECK` with throw `ngraph_error` doesn't avoid this warning.
Other solution would be to move this `NGRAPH_CHECK` to the end of the function (instead of returning empty constant). Would empty `default: break;` be acceptable?

Reply:
@t-jankowski For me it is ok